### ISSUE TITLE
[Proposal] ☂️ Introduce new "Universal design" category

### ISSUE THREAD

**Issue by parlough** [flutter/website#9404]
To replace the current "Accessibility & localization" category in the IA, introduce a new "Universal design" category. Not only will this name allow us to include more types of documents in there, it reflects what accessibility and internationalization are/can be: making things better for everyone universally.

Some steps that should be done as part of this change:

- [ ] Breakup "Internationalizing Flutter apps" doc into multiple, streamlined docs. Perhaps as a new subcategory.
- [ ] Some topics discussed on the Accessibility page should likely become standalone pages.

As follow-up to improve the section, some potential new topic to discuss are as follows:

- [ ] Limit animations/motion when requested
- [ ] Support right to left text and UI
- [ ] Test your app's accessibility
- [ ] Supporting voice control
